docs/conf/scalar-prometheus-custom-values.yaml

```
@@ -26,6 +26,13 @@ grafana:
     # requests:
     #   memory: 400Mi
 
+  # -- Grafana's primary configuration
+  grafana.ini:
+    security:
+      # -- allow Grafana to be embedded (not set the X-Frame-Options header)
+      # -- https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding
+      allow_embedding: true
```

```suggestion
      # -- allow Grafana to be embedded (not set the X-Frame-Options header)
      # -- If you use Scalar Manager, you need to set allow_embedding to true.
      # -- https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding
      allow_embedding: false
```
According to the following doc, this configuration mitigates the risk of Clickjacking. 
https://grafana.com/docs/grafana/latest/administration/configuration/#allow_embedding

Also, I think we use this sample custom values file in several cases other than Scalar Manager. 
And, we don't need to set `true` when we don't use Scalar Manager.

So, I think it is better to set `false` by default and set `true` when we use Scalar Manager only.
I think we can add a comment that we need to set `true` when we use Scalar Manager in the sample custom values file.

What do you think?

---

@kota2and3kan 
Thanks for your feedback.
I agree with your concerns.
It should be set with the default value `false` in the sample custom values.
I will set it `true` as long as we really need it.